### PR TITLE
Fix/doc 4308

### PR DESCRIPTION
**Pull request by mzorai**

### CODE REVIEW

File: docs/Gamification.rst [hunk 1]
```diff
@@ -28,11 +28,61 @@ Domains
...
+|image5|
+
+In this interface, you have two tabs: ``MyPoints`` and ``MyBadges``.
+
+.. _MyPoints:
+
+In this view, you can see these elements:
+
+-  your total points
+-  a table listing the events which allowed you to gain points. This table is composed by these fields:
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
-  a table listing the events which allowed you to earn points. This table is composed by these fields:
```

File: docs/Gamification.rst [hunk 2]
```diff
@@ -28,11 +28,61 @@ Domains
...
+
+   -  The user avatar with whom you or he reacted
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
   -  The user involved in the interaction
```

File: docs/Gamification.rst [hunk 3]
```diff
@@ -28,11 +28,61 @@ Domains
...
+   -  The event you or another user made and which allowed you to gain points
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
   -  The event you or another user made and which allowed you to earn points
```

File: docs/Gamification.rst [hunk 4]
```diff
@@ -28,11 +28,61 @@ Domains
...
+   -  The date of the event
+   -  The set of points you earned following the previously cited event
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
   -  The amount of points you earned following the previously cited event
```

File: docs/Gamification.rst [hunk 5]
```diff
@@ -28,11 +28,61 @@ Domains
...
+   -  The domain of the event
+
+|image6|
+
+.. _MyBadges: 
+
+In the ``MyBadges`` view, you can see all the available badges in each domain with statuses:
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
In the ``MyBadges`` screen, you can see all the badges available in each domain with statuses:
```

File: docs/Gamification.rst [hunk 6]
```diff
@@ -28,11 +28,61 @@ Domains
...
+
+-  *Enabled status*: when the badge is fully colored which means that you already earned this badge
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
-  *Acquired*: when the badge is fully colorized it means that you already earned this badge
```

File: docs/Gamification.rst [hunk 7]
```diff
@@ -28,11 +28,61 @@ Domains
...
+-  *Disabled status*: when the badge is fully greyed which means that you did not yet earned the badge
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
-  *Not acquired*: when the badge is fully greyed, it means that you did not earn the badge yet
```

File: docs/Gamification.rst [hunk 8]
```diff
@@ -28,11 +28,61 @@ Domains
...
+-  *Half enabled status*: When the badge is partially colored which means that your earned points in the
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
-  *in progress*: When the badge is partially colorized it means that you started earning points in the 
```

File: docs/Gamification.rst [hunk 9]
```diff
@@ -28,11 +28,61 @@ Domains
...
+|image6|
+
+.. _MyBadges: 
+
+In the ``MyBadges`` view, you can see all the available badges in each domain with statuses:
+
+-  *Enabled status*: when the badge is fully colored which means that you already earned this badge
+-  *Disabled status*: when the badge is fully greyed which means that you did not yet earned the badge
+-  *Half enabled status*: When the badge is partially colored which means that your earned points in the 
+   domain of that badge are between the  minimum and the maximum points of that badge.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
   domain of that badge, but you've not yet reached the required threshold. 
```

File: docs/Gamification.rst [hunk 10]
```diff
@@ -28,11 +28,61 @@ Domains
...
+
+.. _MyBadges: 
+
+In the ``MyBadges`` view, you can see all the available badges in each domain with statuses:
+
+-  *Enabled status*: when the badge is fully colored which means that you already earned this badge
+-  *Disabled status*: when the badge is fully greyed which means that you did not yet earned the badge
+-  *Half enabled status*: When the badge is partially colored which means that your earned points in the 
+   domain of that badge are between the  minimum and the maximum points of that badge. 
+   For example when the score of a badge is comprised between 500 and 1200 and that your score in the domain is 700.
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
   For example, when the required score of a badge is between 500 and 1200 and that your score in the domain is 700.
```

File: docs/Gamification.rst [hunk 11]
```diff
@@ -28,11 +28,61 @@ Domains
...
+   
+|image7|
+
+When you mouse over a badge, its information appear in a pop up:
+
+-  Level
+-  Name
+-  Description
+-  The badge's points area
```
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
-  The badge's domain
```

**Review by plamarque — CHANGES_REQUESTED** [review]
typos and language suggestions

**Review by hghorbel — APPROVED** [review]

**Review by hghorbel — APPROVED** [review]

**Comment by mzorai** [comment]
@plamarque  we need your approval to validate this PR

**Review by plamarque — APPROVED** [review]
LGTM

**Pull request merged**